app/views/layouts/account_side_nav.html.erb

```diff
@@ -29,11 +29,11 @@
     <div class="display-none tablet:display-block tablet:grid-col-4">
       <%= render 'accounts/side_nav' %>
     </div>
-    <div class="tablet:grid-col-8">
+    <main class="tablet:grid-col-8" id="main-content">
       <%= yield %>
-    </div>
+    </main>
   </div>
 <% end %>
 
 <%= javascript_packs_tag_once('navigation') %>
-<%= render template: 'layouts/base', locals: { disable_card: true } %>
+<%= render template: 'layouts/base', locals: { disable_card: true, custom_main: true } %>
```

Comment:
what if we named the argument `use_main` to indicate that the template should *not* have a `<main>` tag? (and flip the corresponding logic)

```suggestion
<%= render template: 'layouts/base', locals: { disable_card: true, use_main: false } %>
```

Reply:
another idea: `user_main_tag: false` ?

Reply:
I've just realized a different approach or more changes are necessary, as the rule https://dequeuniversity.com/rules/axe/4.3/region?application=axeAPI is violated.

A simple change would be to change how the line `<%= link_to t('shared.skip_link'), '#main-content', class: 'usa-skipnav' %>` works in the base layout. The id of `#main-content` could be changed by the layout file `identity-idp/app/views/layouts/account_side_nav.html.erb`.

Otherwise a bigger restructure of the layouts and accounts page seems to be necessary. The partial `identity-idp/app/views/accounts/_side_nav.html.erb` functions as a second `nav`. But this partial is nested in a `main` tag.

Reply:
For the accessibility build failure, it looks like the only content which is not within a landmark is the top "Access your government benefits ..." content. It seems like it would be appropriate to mark this up as an [`<aside>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside), since it's content which complements the main content?

i.e. changing this element from `<div>` to `<aside>`:

https://github.com/18F/identity-idp/blob/92c73f144e4c96fcc382dc16be79c9eac186cae6/app/views/layouts/account_side_nav.html.erb#L10

Reply:
@zachmargolis I have made your suggested changes. But I find the name `user_main_tag: false` to be confusing, and I find the boolean check for false more difficult to read.